crowdsource/issues.py

```diff
@@ -0,0 +1,8 @@
+from dataquality.issues import IssueType
+
+# Issues with related_class "community-person"f
```

Comment:
?

Reply:
We need new related_class, as we don't want the issues created by users to be deleted the way, the issues created automatically gets deleted.